src/webapp/components/calls-history/CallsHistoryContent.tsx

```diff
@@ -1,30 +1,17 @@
 import styled from "styled-components";
-import { useAppContext } from "../../contexts/app-context";
-import { CircularProgress, Typography } from "@material-ui/core";
 import { CallsTable } from "./CallsTable";
 import { useLocation } from "react-router-dom";
 import { data } from "./mock-tables-data.json";
```

Comment:
Mock Json in used directly from a component. We should create hook, use case an repository